docs/api.md

```diff
@@ -0,0 +1,15 @@
+# auro-dropdownbib
```

Comment:
this needs to be under `components/dropdown/docs`, I assume?

Reply:
This will be addressed here:

-https://github.com/AlaskaAirlines/auro-form/issues/30